### PR TITLE
Initialize workflow cookbook

### PR DESCRIPTION
**Pull request by jmleroux**
diagrams made with gliffy chrome extension

### CODE REVIEW

**Comment by pierallard** [comment]
- cookbook/workflow/img/partial-approve.png : last green box truncates value
- on the 2 images, "creates a draft" > "create a draft" (or change all other ones)

GTM after this comments.
Maybe you should include the origin files to allow future updates.

File: cookbook/workflow/partial_workflow.rst [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Partial workflow
+================
+
+Julia approves or reject a draft partially
```
Comment (jjanvier) [inline cookbook/workflow/partial_workflow.rst]: rejectS

File: cookbook/workflow/simple_workflow.rst [hunk 2]
```diff
@@ -0,0 +1,6 @@
+Simple workflow
+===============
+
+Julia approves or reject a whole draft
```
Comment (jjanvier) [inline cookbook/workflow/simple_workflow.rst]: rejectS

**Comment by jjanvier** [comment]
agree with Pierre to include the origin or SVG files

GTM once fixed

**Comment by rybus** [comment]
On your image: you should write pariatial "rejection/removal"

**Pull request merged**